dcs_core/integrations/databases/sybase.py

```
@@ -149,24 +147,15 @@ def connect(self) -> Any:
                         self.connection = pyodbc.connect(**final_config)
                         logger.info(
-                            f"Successfully connected to Sybase using: "
-                            f"driver={driver}, "
-                            f"{attempt['key']}={attempt['value']}, "
-                            f"port_config={port_config}, "
-                            f"ase_config={ase_config}"
+                            "Successfully connected to Sybase using: "
+                            f"driver={driver}"
                         )
                         return self.connection
                     except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)